pandas/tests/series/test_operators.py

```diff
@@ -595,77 +595,85 @@ def test_divide_decimal(self):
 
         assert_series_equal(expected, s)
 
-    def test_div(self):
+    @pytest.mark.parametrize('dtype2', [np.int64, np.int32, np.int16, np.int8,
```

Comment:
wouldn't this be better as a fixture

Reply:
Maybe eventually.  ATM it is only used once so there's no upside.

Reply:
as a style note, generally I would write this like

```
@pytest.mark.parameterize(
    'dtype2',
     [
        np.int64, ......])
```
as it makes it shorter and more readable

Reply:
i'd like this changed